### PR TITLE
Fix translations

### PR DESCRIPTION
**Pull request by nickdnk**
Followup to https://github.com/splewis/get5/pull/962.

### CODE REVIEW

**Comment by Iwhite67** [comment]
L49 : une fois que la GOTV
I miss : la

**Comment by nickdnk** [comment]
@Iwhite67 Should be good now. I also changed all instances of "Taper" to "Tapez".

File: translations/fr/get5.phrases.txt [hunk 1]
```diff
@@ -182,7 +182,7 @@
     }
     "TeamIsReadyForMapSelection"
     {
-        "fr"            "{1} est prêt pour le tour de veto."
+        "fr"            "{1} est prête pour la sélection de map."
```
Comment (nickdnk) [inline translations/fr/get5.phrases.txt:185]: @Iwhite67 This was changed to "prête" over "prêt". Is this intentional? Note that the first word here is a variable; the team's name.

**Comment by Apfelwurm** [comment]
German:
```
Type [!ready] when your team is ready for map selection. (!ready is a variable)
Tippe {1} wenn dein team bereit für die Kartenauswahl ist.

The map will change once GOTV has broadcast the map selection.
Die Karte wird gewechselt sobald die Kartenauswahl von GOTV übertragen wurden.

[TeamA] is ready for map selection. (TeamA is variable)
{1} ist bereit für die Kartenauswahl.

[TeamA] banned [de_nuke]. (TeamA and de_nuke are variables)
{1} legte ein veto gegen die Karte {2} ein.

A team captain left during map selection. Map selection is paused.
Der Team Captian hat das Spiel während der Kartenauswahl verlassen, daher wird die Kartenauswahl pausiert.

Type [!ready] when you are ready to resume map selection. (!ready is a variable)
Tippe {1} wenn ihr bereit seid mit der Vetophase fortzufahren.

Select a map to BAN:
Wähle eine Karte aus gegen die ein VETO eingelegt werden soll:


Confirm you want to BAN [de_nuke]: (de_nuke is variable)
Bestätige dass ein VETO gegen {1} eingelegt werden soll:

Map selection commencing in [5]... (4,3,2,1 - variable) 
Die Kartenauswahl beginnt in {1} Sekunden.
```




Additionally we should also change the following in this case also:
```
  "WaitingForGOTVBroadcastEnding"
    {
        "de"            "Die Karte wird gewechselt sobald die GOTV Übertragung beendet ist."
    }

 "NextSeriesMapInfoMessage"
    {
        "de"            "Die nächste Karte in der Serie ist {1} und wird in {2} Sekunden starten."
    }

    "ChangingMapInfoMessage"
    {
        "de"            "Ändern der Karte in {1}..."
    }

    "MapDecidedInfoMessage"
    {
        "de"            "Folgende Karten wurden ausgewählt:"
    }

    "MapIsInfoMessage"
    {
        "de"            "Karte {1}: {2}"
    }

    "TeamPickedMap"
    {
        "de"            "{1} wählte {2} als {3} Karte aus."
    }
```

**Pull request merged**